packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -120,7 +121,7 @@ class CircleAvatar extends StatelessWidget {
         child: new MediaQuery(
           // Need to reset the textScaleFactor here so that the
           // text doesn't escape the avatar when the textScaleFactor is large.
-          data: const MediaQueryData(textScaleFactor: 1.0),
+          data: MediaQuery.of(context, nullOk: true).copyWith(textScaleFactor: 1.0),
```

Comment:
if you say nullOk, you're going to get back null... which will promptly crash since you dereference it.
I think you want to remove nullOk.

Reply:
Duh. Yes, that was an oversight.